src/components/common/KeyboardAreaView/index.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { KeyboardAvoidingView, Platform, ScrollView } from 'react-native';
+import { object, element } from 'prop-types';
+import styles from './styles';
+
+const KeyboardAreaView = ({ children, styleContainer, styleContentScrollView, ...props }) => {
+  return (
+    <KeyboardAvoidingView
+      behavior={Platform.OS === 'ios' ? 'padding' : 'height'}
```

Comment:
A suggestion it is very common to use this validation you could create a constant in your file for general constants 

for example
export const isIOS = Platform.OS === 'ios';